LibertyDSNP/contracts
# Add dev tools

Problem
=======
Some useful dev tools are missing, we should add them
[link to Pivotal Tracker #178004218](https://www.pivotaltracker.com/story/show/178004218)

Solution
========
Added dev tools

Change summary:
---------------
* Added eth-gas-reporter for gas reports from tests
* Added .mocharc.json so IDEA and VS Code (and others?) can run tests in the IDE

Steps to Verify:
----------------
1. Can run tests in IDEA or VS Code
2. Can see gas report after running npm tun test
